tests/test_nanoleaf_client.py

```
@@ -251,6 +337,98 @@ def test_stream_drops_when_interval_has_not_elapsed(self):
+            streamed.append(list(panels))
+
+        with patch.object(client, 'ensure_layout', return_value=device), patch.object(client, 'prepare_streaming'), patch.object(client, '_stream', side_effect=capture), patch('nanoleaf_client.time.sleep'):
+            client.identify_panel(device, 11, wait=True)
+        self.assertEqual(streamed[-1], prior)
+        self.assertTrue(device.ext_control_active)
+
+    def test_identify_panel_restores_rest_state_without_prior_frame(self):
+        from nanoleaf_client import NanoleafClient, NanoleafDevice
+        client = NanoleafClient()
+        self.addCleanup(client.close)
+        device = NanoleafDevice('nl_x', '127.0.0.1', auth_token='tok', model='NL59')
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)